atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/render/TableRenderer.java

```diff
@@ -866,12 +873,22 @@ private void renderNamedDelete() {
             } line("}");
         }
 
-        private void renderGetRange() {
-            line("public BatchingVisitableView<", RowResult, "> getRange(RangeRequest range) {"); {
+        private void renderOptimizeRangeRequests() {
+            line("private RangeRequest optimizeColumnSelection(RangeRequest range) {"); {
```

Comment:
I think we still want this to be named `optimizeRangeRequest`? A `ColumnSelection` is a different thing (which I see you've added renderer code for later on).